### DISCUSSION TITLE
ReferenceError: FileList is not defined

### DISCUSSION THREAD

**Discussion by IRediTOTO** [blitz-js/blitz#3017] [Q&A] [answered]
Hi, i have a file upload field in form. I am trying to use it with Zod, I added `dom` to tsconfig file
everything look fine, but they thrown me an error like this. Is there something wrong here? 
![image](https://user-images.githubusercontent.com/15695252/144698686-b7e18703-01d8-4172-972a-6bff958aecd3.png)

**Comment by beerose** [comment, marked as answer]
Is `FileList` imported into this file? This error means that in this file, it can't find anything called `FileList`.
